### PR TITLE
Publish CC release 11 April

### PR DESCRIPTION
**Pull request by ctauchen**
Product Version(s):

Issue:

Link to docs preview:

https://deploy-preview-561--tigera.netlify.app/calico-cloud/release-notes/

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -406,11 +406,11 @@ const config = {
           },
           3.16: {
             label: '3.16',
```
Comment (tmjd) [inline docusaurus.config.js]: 3.15 did not have a label, does it matter that v3.16 does?

Comment (ctauchen) [reply docusaurus.config.js]: Nope, doesn't get used unless multiple versions are being published. But I've removed it anyway. Thanks!

File: calico-cloud_versioned_docs/version-3.16/release-notes/index.mdx [hunk 2]
```diff
@@ -4,6 +4,52 @@ description: What's new, and why features provide value for upgrading.
 
 # Release notes
```
Comment (ctauchen) [inline calico-cloud_versioned_docs/version-3.16/release-notes/index.mdx:5]: @tmjd See anything that doesn't belong?

File: docusaurus.config.js [hunk 3]
```diff
@@ -397,20 +397,19 @@ const config = {
...
             label: 'Next',
             path: 'next',
             banner: 'unreleased',
           },
           3.16: {
-            label: '3.16',
-            path: '3.16',
-            banner: 'unreleased',
+            path: '/',
+            banner: 'none',
```
Comment (doucol) [inline docusaurus.config.js:409]: Taking the `label` field out of here will likely break the switch-latest script

Comment (doucol) [reply docusaurus.config.js:409]: And I think it needs to be the first field in the list.

**Review by doucol — CHANGES_REQUESTED** [review]

**Review by doucol — APPROVED** [review]

**Pull request merged**